internal/pkg/cli/pipeline_init.go

```diff
@@ -94,6 +95,7 @@ type initPipelineOpts struct {
 	initPipelineVars
 	// Interfaces to interact with dependencies.
 	workspace      wsPipelineWriter
+	wsLister       wsPipelineLister
```

Comment:
Instead of adding an additional field, should we update `workspace` to have a bigger interface?
```go
workspace wsPipelineIniter

type wsPipelineIniter interface {
  ... // all workspace pipeline methods to initialize a pipeline.
}
```

Reply:
Either this works or we can use `ListPipelines()` from `wsPipelineGetter`

Reply:
i'll take this approach since `pipeline init` is the only place that currently uses `wsPipelineWriter`